src/huggingface_hub/inference/_generated/_async_client.py

```diff
@@ -149,6 +151,7 @@ def __init__(
         headers: Optional[Dict[str, str]] = None,
         cookies: Optional[Dict[str, str]] = None,
         proxies: Optional[Any] = None,
+        trust_env: bool = True,
```

Comment:
Breaking change, default value is False

```suggestion
        trust_env: bool = False, # Default value False documented here https://github.com/aio-libs/aiohttp/blob/v3.10.0/docs/client_reference.rst?plain=1#L195 
```

Reply:
Yes, thank you, error during testing.